pkg/plugins/platform/router.go

```diff
@@ -96,6 +111,7 @@ func StartDdevRouter() error {
 
 // PrintRouterStatus outputs router status and warning if not
 // running or healthy, as applicable.
+// An easy way to make these ports unavailable: sudo netcat -l -p 80  (brew install netcat)
```

Comment:
this comment seems out of place.

Reply:
It's actually intended here, but I added to it "In order to test this". The comment was to remind anybody how you'd test the port being occupied without a major effort of setting up apache or something.